samples/cors/ts/README.md

```diff
@@ -0,0 +1,105 @@
+# Using Azure Key Vault in a Web Application
+
+Browser security prevents a web page from making requests to a different domain than the one that served the web page. This restriction is called the same-origin policy. The same-origin policy prevents a malicious site from reading sensitive data from another site. Sometimes, you might want to allow other sites to make cross-origin requests to your app. That's where Cross Origin Resource Sharing (CORS) policies become necessary.
+
+While CORS is configurable for some Azure services, Azure Key Vault does not currently support CORS natively. So what can you do if you'd like to integrate with Azure Key Vault from a client side web application?
+
+Fortunately, there are a few options:
+
+- Use a back end server to route requests to Azure Key Vault.
+- Use [Azure API Management][azureapimanagement] to route requests to Azure Key Vault.
+
+> Rememeber: CORS is a _browser_ restriction, and is not a concern for Node applications.
+
+Interested in enabling CORS in Key Vault? [Let us know!](https://feedback.azure.com/forums/906355-azure-key-vault/suggestions/34753195-enable-cors-for-key-vault)
+
+## Use a back end server
```

Comment:
Another fairly low-cost method would be to use Azure Functions, which is super easy to setup and allows you to create a ton of endpoints and triggers right out of the box. 

The nice thing is that, with consumption plans, they're super cheap for experimenting and don't require you to do much planning.

Reply:
Thanks! I added a note for myself to investigate that option in the very near future!